content/en/docs/faq/observability/byop.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: "Bring Your Own Prometheus"
+title: "Install Your Own Prometheus"
```

Comment:
Just my two cents:

I would like to confirm here. In fact `Bring Your Own (BYO)` is often used in IT, such as Bring Your Own Software. You can also find similar expressions in [Google Doc](https://cloud.google.com/vpc/docs/using-bring-your-own-ip). The title is to tell readers that they can integrate external Prometheus into KubeSphere. So can we continue to use `Bring Your Own Prometheus`?

Reply:
Ok